app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -209,6 +209,7 @@ abstract class RoutePresenter(
         }
     }
 
+    @Suppress("DEPRECATION") // Fixing the deprecation requires significant changes.
```

Comment:
Do we know when this is going to be removed? I hope we never have to find out, but would be good to add to your comment.

Reply:
We would only have to address if we move to a newer `targetVersion` that no longer has this API and I think we'd have enough time by then to decide at that point.